### ISSUE TITLE
Typo error for blog.

### ISSUE THREAD

**Issue by tracer9** [arunmallya/arunmallya.github.io#1]
Hi arunmallya:

Recently I found your personal website and I learned a lot form your blog, thanks!

I found a typo error in your post "A Quick Introduction to Backpropagation". In the section
"Softmax Layer", the back propagation seems to be this, you missed an item. 

![m](https://cloud.githubusercontent.com/assets/6302380/17587398/a8f365f2-5ffa-11e6-837d-ac43ef8db5d3.png)

Your final result is all right, so I think it is just a type error.

**Comment by arunmallya** [comment]
Thanks for letting me know. Will fix.

**Comment by arunmallya** [comment]
Finally fixed!

Thank you!
